src/explorer/gql/utils.ts

```diff
@@ -0,0 +1,27 @@
+export const BRIDGE_TXN_API_GRAPHQL_URL =
+  "https://syn-explorer-api.metagabbar.xyz/graphql";
+
+export const QUERY_INFO = {
```

Comment:
also, can add more queries here easily to set specific data types that users want to get returned back